File: Programming-languages-and-laboratory/Python/Exercises/37_compito_python_20200124/Squash.py
from Campo import Campo


class Squash(Campo):
    def __init__(self, codice, sport, nome, larghezza, lunghezza, altezza, piano, costo):
        super().__init__(codice, sport, nome, larghezza, lunghezza, costo)
        self._altezza = altezza
        self._piano = piano

    def __str__(self):
        return self._sport + "\t" + self._nome + "\t" + str(self._codice) + "\t" + str(self._larghezza) + "\t" +\
               str(self._lunghezza) + "\t-\t-\t" + "\t" + str(self._altezza) + "\t" + str(self._piano) + "\t" +\
               str(self._costo)

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/37_compito_java_2020_01_24/Squash.java
public class Squash extends Campo {
	private float altezza;
	private int piano;
	private float costo;

	public Squash(int codice, String sport, String nome, float larghezza, float lunghezza, float altezza, int piano, float costo) {
		super(codice, sport, nome, larghezza, lunghezza, costo);
		this.altezza = altezza;
		this.piano = piano;
	}

	public String toString() {
		return sport + "\t" + nome + "\t" + codice + "\t" + larghezza + "\t" + lunghezza + "\t-\t-\t" + altezza + "\t"
				+ piano + "\t" + costo;
	}
}